src/sage/combinat/integer_vectors_mod_permgroup.py

```diff
@@ -1,4 +1,4 @@
-# optional - sage.combinat sage.groups
+# sage.doctest: needs sage.combinat sage.groups
 r"""
```

Comment:
Remaining doctest warnings in this file:
```
sage -t --long --warn-long 33.8 --random-seed=61144299382637325659043318932957598596 src/sage/combinat/integer_vectors_mod_permgroup.py
**********************************************************************
File "src/sage/combinat/integer_vectors_mod_permgroup.py", line 190, in sage.combinat.integer_vectors_mod_permgroup.IntegerVectorsModPermutationGroup
Warning: Consider using a block-scoped tag by inserting the line 'sage: # long time' just before this line to avoid repeating the tag 4 times
    G = PermutationGroup([[(6,)]]) # long time
    [171 tests, 0.58 s]
```

Reply:
Done in 82fb0f5fec3